app/src/main/java/fr/free/nrw/commons/customselector/ui/adapter/ImageAdapter.kt

```diff
@@ -0,0 +1,91 @@
+package fr.free.nrw.commons.customselector.ui.adapter
+
+import android.content.Context
+import android.view.ViewGroup
+import fr.free.nrw.commons.R
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.constraintlayout.widget.Group
+import androidx.recyclerview.widget.RecyclerView
+import fr.free.nrw.commons.customselector.listeners.ImageSelectListener
+import fr.free.nrw.commons.customselector.model.Image
+
+class ImageAdapter(
```

Comment:
For your recyclerview adapters you can use DiffUtils

Reply:
Done.